components/NavBar/components/NetworksSwitch/NetworksSwitch.tsx

```diff
@@ -1,18 +1,11 @@
 import React, { useContext } from 'react';
 import { INetwork, NetworkContext } from '../../../../contexts/NetworksContext';
```

Comment:
FMI, what is the I standing for in INetwork ? Thanks

Reply:
I prefix is used to mean interfaces, we dont use it that much but it is a handy standard